Update syntax for the saveTableToContext step (#3845)
vividus-framework/vividus: docs/modules/plugins/pages/plugin-web-app.adoc

When I click on a button with the name 'Toggle element visibility with delay'
Then element located 'By.xpath(//div[@id='delayed'])' disappears in 'PT3S'
----

=== Set Variable Steps

==== Save HTML table to the variable

This step allows you to save an HTML table from a specified context into a variable.

> Values are stored in the variable as a list of maps. For more information, see xref:commons:variables.adoc#_working_with_tables[working with table variables]

Reviewer: ```suggestion
For more information, see how xref:commons:variables.adoc#_working_with_tables[to work with table variables]
```

[IMPORTANT]
The HTML table must have a header row with <th> tags.

[source,gherkin]
----
When I save table to $scopes variable `$variableName`
----

* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
* `$variableName` - The variable name to refer to the variable value.

[NOTE]

> Before using this step, ensure the test context is set to the required table. See the example below.

Reviewer: ```suggestion
Before using this step, ensure the context is set to the required table. See the example below.
```
